dotcom-rendering/src/components/FooterReaderRevenueLinks.importable.tsx

```diff
@@ -0,0 +1,154 @@
+import { css } from '@emotion/react';
+import {
+	brandText,
+	from,
+	headlineBold20,
+	headlineBold24,
+	headlineBold28,
+	headlineBold34,
+	palette,
+	textSans15,
+	textSans17,
+	until,
+} from '@guardian/source/foundations';
+import { shouldHideSupportMessaging } from '../lib/contributions';
+import { useCountryCode } from '../lib/useCountryCode';
+import ArrowRightIcon from '../static/icons/arrow-right.svg';
```

Comment:
Could we use [Source’s icons](https://guardian.github.io/storybooks/?path=/docs/source_react-components-icons--docs)? Probably [`SvgArrowRightStraight`](https://github.com/guardian/csnx/blob/44fd7be64c123ea65bcfc532725a6fa766ffa105/libs/%40guardian/source/src/react-components/__generated__/icons/SvgArrowRightStraight.tsx)

Reply:
Good spot! This component was very old, I'll create a health card to update this